client/modules/core/helpers/permissions.js

```diff
@@ -59,6 +59,10 @@ Template.registerHelper("hasDashboardAccess", function () {
   return Reaction.hasDashboardAccess();
 });
 
+Template.registerHelper("hasDashboardAccessForAnyShop", function () {
+  return Reaction.hasPermissionsForAnyShop();
```

Comment:
Good catch. I think I added this before the navbar was fully converted to react and have since removed the rest of it.